src/main/java/uk/gov/hmcts/reform/data/ingestion/camel/processor/FileReadProcessor.java

```diff
@@ -1,30 +1,91 @@
 package uk.gov.hmcts.reform.data.ingestion.camel.processor;
 
+import static java.util.Objects.isNull;
+import static uk.gov.hmcts.reform.data.ingestion.camel.util.MappingConstants.BLOBPATH;
+import static uk.gov.hmcts.reform.data.ingestion.camel.util.MappingConstants.IS_FILE_STALE;
+import static uk.gov.hmcts.reform.data.ingestion.camel.util.MappingConstants.ROUTE_DETAILS;
+
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.blob.CloudBlobClient;
+import com.microsoft.azure.storage.blob.CloudBlobContainer;
+import com.microsoft.azure.storage.blob.CloudBlockBlob;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.camel.CamelContext;
 import org.apache.camel.ConsumerTemplate;
 import org.apache.camel.Exchange;
 import org.apache.camel.Processor;
+import org.apache.commons.lang.time.DateFormatUtils;
+import org.apache.commons.lang.time.DateUtils;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
-import uk.gov.hmcts.reform.data.ingestion.camel.util.MappingConstants;
+import uk.gov.hmcts.reform.data.ingestion.camel.route.beans.RouteProperties;
+import uk.gov.hmcts.reform.data.ingestion.camel.service.AuditServiceImpl;
+import uk.gov.hmcts.reform.data.ingestion.configuration.AzureBlobConfig;
+import java.util.Date;
 
 @Slf4j
 @Component
 public class FileReadProcessor implements Processor {
 
     @Value("${file-read-time-out}")
-    int fileReadTimeOut;
+    protected int fileReadTimeOut;
```

Comment:
Why protected shouldn't be private?Applies for all vars

Reply:
since we are reusing this class in integration test cases to avoid calling blob storage